resources/sts/4.17/hypershift/sts_hcp_worker_instance_permission_policy.json

```diff
@@ -1,13 +1,34 @@
 {
     "Version": "2012-10-17",
     "Statement": [
```

Comment:
AWS will ask for these to be broken into three separate statements, and they will all need explanatory SIDs as well

Reply:
+1 to this other wise its good to go.

Reply:
I've added the Sid statements, let me know if there is anything else to do before we merge.